docs/sources/administration/image_rendering.md

```diff
@@ -43,7 +43,33 @@ For further information and instructions refer to the [plugin details](https://g
 
 The [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can also be run as a remote HTTP rendering service. In this setup Grafana will render an image by making a HTTP request to the remote rendering service, which in turn render the image and returns it back in the HTTP response to Grafana.
 
-To configure Grafana to use a remote HTTP rendering service, please refer to [rendering](/installation/configuration/#rendering) configuration section.
+As an alternative to installing and running the image renderer as a plugin you can run it as a remote image rendering service using Docker.
```

Comment:
Think you can rewrite this to describe that the remote HTTP rendering service can either be run as a node.js app (from source) or in a docker container.